### PR TITLE
More `iproute2` and `dhcpcd` additions

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: options/posix/include/netinet/if_ether.h [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#ifndef _NETINET_IF_ETHER
```
Comment (64) [inline options/posix/include/netinet/if_ether.h:1]: _H

File: options/posix/include/netinet/if_ether.h [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+struct ether_arp {
+	struct arphdr ea_hdr;
+	uint8_t arp_sha[ETH_ALEN];
+	uint8_t arp_spa[4];
+	uint8_t arp_tha[ETH_ALEN];
+	uint8_t arp_tpa[4];
+};
+
+#endif //_NETINET_IF_ETHER
```
Comment (64) [inline options/posix/include/netinet/if_ether.h:105]: _H

**Review by 64 — APPROVED** [review]

File: options/posix/include/net/if_arp.h [hunk 3]
```diff
@@ -0,0 +1,103 @@
+#ifndef  _NET_IF_ARP_H
```
Comment (Dennisbonke) [inline options/posix/include/net/if_arp.h:1]: Double space here.

File: options/posix/include/net/if_arp.h [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+#define  _NET_IF_ARP_H
```
Comment (Dennisbonke) [inline options/posix/include/net/if_arp.h:2]: Double space here.

File: options/posix/include/net/if_arp.h [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+#define ARPHRD_ROSE 270
+#define ARPHRD_X25 271
+#define ARPHRD_HWX25 272
+#define ARPHRD_CAN 280
+#define ARPHRD_PPP 512
+#define ARPHRD_CISCO 513
+#define ARPHRD_HDLC ARPHRD_CISCO
+#define ARPHRD_LAPB 516
+#define ARPHRD_DDCMP 517
+#define ARPHRD_RAWHDLC	518
```
Comment (Dennisbonke) [inline options/posix/include/net/if_arp.h:46]: This should be a space.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Few styling issues.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**